apps/framework-docs/src/pages/moose/olap/model-materialized-view.mdx

```diff
@@ -13,7 +13,27 @@ import { Lightbulb } from "lucide-react";
 
 ## Overview
 
-Materialized views are write-time pipelines. They run a static `SELECT` to transform data from one or more source table(s) to a target destination table. You query the destination table like any other table.
+Materialized views are write-time pipelines. They run a static `SELECT` to transform data from one or more source table(s) to a target destination table. You query the destination table like any other table. The MaterializedView class wraps ClickHouse's `MATERIALIZED VIEW` functionality, letting you model these transformations in code rather than raw SQL.
+
+<BulletPointsCard title="Key differences from other databases" bulletStyle="number" compact bullets={[
```

Comment:
Confusing headline. Seems like its referring to MooseStack at first. Maybe something like:
"Key difference with ClickHouse MVs, vs other databases"